qiskit/ignis/verification/randomized_benchmarking/fitters.py

```diff
@@ -416,8 +415,10 @@ def fit_data(self):
             if alpha_guess < 1.0:
                 fit_guess[1] = alpha_guess
 
-            fit_guess[0] = ((y0 - fit_guess[2]) /
```

Comment:
This was a bug on my part, this should have been `fit_guess[1]` and it's just taking a linear approximation of rb_fit_func to guess the `alpha` parameter

Reply:
No, it's really fit_guess[0]. Once you have guessed fit_guess[1] and fit_guess[2], you extract fit_guess[0].